### DISCUSSION TITLE
Unable to enter a Space in Tag Editor after 20210530 update

### DISCUSSION THREAD

**Discussion by ScottVinzant** [rern/rAudio#355] [Fixed Issues]
@rern, it seems that a "SPACE" character is not able to be added between words when editing tags.
This is also noticed when adding/editing names in the WebRadio section.
Notice the "Classic Rock Florida HD" station name.
Also, I was able to REMOVE a space in the "Mellow Mix" name, but not add it back in.

![Screenshot from 2021-05-30 10-30-53](https://user-images.githubusercontent.com/84992585/120110214-32382d00-c132-11eb-9739-5b0a093d54a6.png)

**Comment by rern** [comment]
Try:
`sed -i '/keyCode/,/radio.active. ).click/ d' /srv/http/assets/js/info.js`
Then refresh the browser.

**Reply by ScottVinzant** [reply]
@rern,
That took care of the tag editing issue.
I was also able to edit the Web Radio station info.
Thanks.
